alternatives.c

```diff
@@ -1290,6 +1290,32 @@ static int listServices(const char *altDir, const char *stateDir, int flags) {
     return 0;
 }
 
+int dirExists(const char *path) {
+    struct stat stats;
+    stat(path, &stats);
```

Comment:
This should probably handle errors (other than ENOENT). (That said of course this codebase doesn't do much proper error handling, I know)